CMakeLists.txt

```
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.16)
+cmake_minimum_required(VERSION 3.19)
```

For Ubuntu Focal 20.4 has 3.16 and we have updated Ubuntu Bionic 18.4 to 3.16 as well. I have not found a suitable PPA for 3.19 for both so I suggest to just revert the line and ignore the cpack requirement for MacOs. 
https://packages.ubuntu.com/search?keywords=cmake&searchon=names&suite=all&section=all
```suggestion
cmake_minimum_required(VERSION 3.16)
```
In the main branch we can use 3.19 once we have advanced our minimum to Ubuntu Jammy 22.04 which has 3.22.

---

Mhmm. So do you suggest to only merge this to main and disabled the packaging step for 2.3? But how will we create new 2.3.x builds when we can't package on CI?

---

CPack will just work only because the installed version on MacOS  is >= 3.19 
This is independent from the version guard here. 

This line must be kept as 3.16 to not force failing legacy Ubuntu builds. 
That's all.

---

Reverted the minimum_version change. Was that your request?

---

Yes, just the proposed one liner.

---

Great I'll force push